src/export/utils/walkArticle.ts

```diff
@@ -59,14 +58,19 @@ function getFigureHTML(
 </figure>`;
 }
 
-function getEditorStateFromFirstHTMLOutput(version: Version<Blocks.Output>) {
-  return version.data.outputs.reduce<EditorState<any> | null>((state, { kind, content }) => {
-    if (state != null) return state;
-    if (kind === OutputSummaryKind.html && content) {
-      return getEditorStateFromHTML(content);
-    }
-    return null;
-  }, null);
+async function getEditorStateFromFirstHTMLOutput(version: Version<Blocks.Output>) {
+  // find first
+  const htmlOutput = version.data.outputs.find(
+    (output) => output.kind === OutputSummaryKind.html && Boolean(output.content),
+  );
+  if (!htmlOutput) return null;
+  let { content } = htmlOutput;
+  if (htmlOutput.link) {
+    const response = await fetch(htmlOutput.link);
+    if (!response.ok) return null;
+    content = await response.text();
+  }
```

Comment:
@dylangrandmont this was a gotcha with our HTML outputs that might be relevant to what you are doing. If the `content` of an output gets over a certain size, we truncate it and stach the whole thing on storage... so if a `link` field is present you'll need to fetch the html

Reply:
Thanks for flagging, I definitely have encountered this